tools/travis-test.sh

```diff
@@ -94,16 +94,22 @@ export PIP
 if [ -n "$USE_WHEEL" ] && [ $# -eq 0 ]; then
   # Build wheel
   $PIP install wheel
+  # ensure that the pip / setuptools versions deployed inside the venv are recent enough
+  $PIP install -U virtualenv
   $PYTHON setup.py bdist_wheel
   # Make another virtualenv to install into
-  virtualenv --python=python venv-for-wheel
+  virtualenv --python=`which $PYTHON` venv-for-wheel
   . venv-for-wheel/bin/activate
   # Move out of source directory to avoid finding local numpy
   pushd dist
-  $PIP install --pre --no-index --upgrade --find-links=. numpy
-  $PIP install nose
+  pip install --pre --use-wheel --no-index --upgrade --find-links=. numpy
+  pip install nose
```

Comment:
``--no-use-wheel`` is deprecated in pip 8.0, and ``--use-wheel`` isn't even listed in the help text of ``pip install`` anymore. So I wouldn't add it.